website/src/css/custom.css

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
We need the copyright header in this file as well:

```
/**
 * Copyright (c) 2017-present, Facebook, Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
```

Reply:
done